src/java/com/puppetlabs/http/client/Sync.java

```diff
@@ -86,8 +87,17 @@ private static Response request(SimpleRequestOptions simpleRequestOptions,
      * @return A persistent synchronous HTTP client
      */
     public static SyncHttpClient createClient(ClientOptions clientOptions) {
-        return new PersistentSyncHttpClient(
-                JavaClient.createClient(clientOptions));
+        return createClient(clientOptions, null);
+    }
+
+    /**
+     * Creates a synchronous persistent HTTP client
+     * @param clientOptions the list of options with which to configure the client
+     * @param metricRegistry a metric registry to track metrics on client requests
+     * @return A persistent synchronous HTTP client
+     */
+    public static SyncHttpClient createClient(ClientOptions clientOptions, MetricRegistry metricRegistry) {
```

Comment:
Same comment here as with the Clojure side implementation.  I think it would be good to have a way to pass in a `MetricRegistry` even for one-off Sync requests like Sync.get().  In this case, we could provide that via another one of the `SimpleRequestOptions` or overload the existing methods with a new version that takes a `MetricRegistry` object.